src/Widgets/DiskBar.vala

```diff
@@ -174,7 +180,7 @@ public class Installer.DiskBar: Gtk.Grid {
                 // Begin by resizing all partitions over 20px wide.
                 bool excess_modified = false;
                 for (int y = 0; excess > 0 && y < x; y++) {
-                    if (lengths[y] <= 20) continue;
+                    if (lengths[y] <= 28) continue;
```

Comment:
As long as we're modifying this, does it make sense to use a CONST to keep it all in sync?

Reply:
Sure, I'll do that.